spec/factories/animals.rb

```diff
@@ -13,17 +12,15 @@
     collection_year { 1 }
     date_time_collected { "2020-04-04 18:52:09" }
     collection_position { "MyString" }
-    pii_tag { 1 }
-    tag_id { 1 }
+    sequence(:tag) { |n| "Y#{n}" }
     sex { 'male' }
     organization
   end
   factory :female, class: Animal do
     collection_year { 1 }
     date_time_collected { "2020-04-04 18:52:09" }
     collection_position { "MyString" }
-    pii_tag { 1 }
-    tag_id { 1 }
+    sequence(:tag) { |n| "R#{n}" }
```

Comment:
Same suggestion here about left-padding the tag... 
```suggestion
    sequence(:tag) { |n| "R#{sprintf('%03d', 1)}" }
```